sql_upgrade.php

```diff
@@ -110,7 +110,7 @@
 foreach ($versions as $version => $filename) {
   echo " <option value='$version'";
   // Defaulting to most recent version, which is now 4.2.1.
```

Comment:
bad comment, refers to old version #.  may change comment to say forked from version 4.2.1 or drop it